### DISCUSSION TITLE
Deploy Bridge as external Loadbalancer service

### DISCUSSION THREAD

**Discussion by vijayrajah** [strimzi/strimzi-kafka-operator#4757] [Q&A] [answered]
In the current config (using latest operator version --0.22.1) the kafka bridges' HTTP port is configured as cluster IP.
 
I'm looking to change the HTTP port to external loadbalancer type (with annotations). Is this possible?

**Comment by scholzj** [comment, marked as answer]
You can create your own service with your own name and use any type you want. Just make sure to use different name to avoid conflict with the operator.

**Reply by vijayrajah** [reply]
Yes.. Perhaps I should have clarified.. Sorry about that

I have created a custom service with same selectors.

I wanted to check if this is natively supported by the operator without creating the resource myself.

**Reply by scholzj** [reply]
Ahh, sorry ... I didn't answered that part directly. It is not supported right now, the service created by the operator is always ClusterIP type.

**Reply by vijayrajah** [reply]
@scholzj - Thanks for you feedback. I have raised a issue https://github.com/strimzi/strimzi-kafka-operator/issues/4759

**Reply by scholzj** [reply]
I do not think we plan to support this right now. The HTTP interface is not secured in any way right now and it is expected to put behind some API gateway, authentication proxy etc. to be secured. Exposing the unsecured HTTP endpoint to the outside of the cluster makes only little sense. So if we would want to implement this, I think the security needs to come first.
